### PR TITLE
Quality control for human biomolecular data page

### PR DESCRIPTION
**Pull request by lauportell**

### CODE REVIEW

File: quality-control/human-biomolecular-data.md [hunk 1]
```diff
@@ -1,27 +1,98 @@
...
+- {% tool "picard" %}: Picard is a suite of tools that provides quality control and processing of NGS data, including duplicate read removal, format conversion, and alignment. It is widely used in sequencing data analysis pipelines to improve the quality and reliability of the data.
+- {% tool "trimmomatic" %}: Trimmomatic is a tool used for the removal of adapter sequences, low-quality reads, and sequences with ambiguous bases from NGS data. It uses various filtering and trimming options to improve the quality of the sequencing data and prepare it for downstream analysis.
+- {% tool "fastqc-screen" %}:  FastQScreen is a quality control tool used to detect contamination in sequencing data (FASTQ files). It screens the input data against a database, formed by the genomes of all of the organisms that could have contaminated the sample, along with PhiX, Vectors or other contaminants commonly seen in sequencing experiments.
+- {% tool "graf-pop" %}: GRAF pop is a software tool that infers the subject ancestry. For the estimation it uses about 100,000 SNPs from genotype datasets, which can be in PLINK and VCF format.
```
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:71]:
```suggestion
- {% tool "graf-pop" %}: GRAF-pop is a software tool that infers the subject ancestry. For the estimation it uses about 100,000 SNPs from genotype datasets, which can be in PLINK and VCF format. 
```

File: quality-control/human-biomolecular-data.md [hunk 2]
```diff
@@ -1,27 +1,98 @@
...
+- {% tool "trimmomatic" %}: Trimmomatic is a tool used for the removal of adapter sequences, low-quality reads, and sequences with ambiguous bases from NGS data. It uses various filtering and trimming options to improve the quality of the sequencing data and prepare it for downstream analysis.
+- {% tool "fastqc-screen" %}:  FastQScreen is a quality control tool used to detect contamination in sequencing data (FASTQ files). It screens the input data against a database, formed by the genomes of all of the organisms that could have contaminated the sample, along with PhiX, Vectors or other contaminants commonly seen in sequencing experiments.
+- {% tool "graf-pop" %}: GRAF pop is a software tool that infers the subject ancestry. For the estimation it uses about 100,000 SNPs from genotype datasets, which can be in PLINK and VCF format. 
+- {% tool "graf-sex" %}: Determines subject sexes using the genotypes. It’s used to validate the self-reported sexes in phenotype datasets.
```
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:72]:
```suggestion
- {% tool "graf-sex" %}: GRAF-sex determines subject sexes using the genotypes. It’s used to validate the self-reported sexes in phenotype datasets.
```

File: quality-control/human-biomolecular-data.md [hunk 3]
```diff
@@ -1,27 +1,98 @@
...
+- **Procedural documents**:  How an speciﬁc laboratory activity is conducted or how a piece of equipment is has been used must be recorded in standard operating procedures (SOPs) and equipment operating procedures (EOPs). These records will ensure results reproducibility. It’s also important to notice that these records must undergo regular reviews, ensuring that they remain compliant.
+- **Negative and positive controls**: including negative controls it’s important to detect contamination in the reagents, consumables, and general laboratory environment. Including negative controls in each run of diagnostic molecular testing based on RT-PCR, for example, will avoid false positive results resulting from cross-contamination. As well as positive controls for the target sequence/s used in the individual assays, to ensure no false negative results due to reagents failure. For example, the study of a housekeeping human gene would ensure quality of the sample collection procedure: it controls that the sample has human material and quality of the analytical procedure: we check there is no PCR inhibition and the reagents are working properly.
```
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:80]:
```suggestion
- **Negative and positive controls**: including negative controls it’s important to detect contamination in the reagents, consumables, and general laboratory environment. Including negative controls in each run of diagnostic molecular testing based on RT-PCR, for example, will avoid false positive results resulting from cross-contamination. As well as positive controls for the target sequence/s used in the individual assays, to ensure no false negative results due to reagents failure. For example, the study of a housekeeping human gene would ensure quality of the sample collection procedure, as it controls that the sample has human material and the quality of the analytical procedure, making it possible to check that there is no PCR inhibition and the reagents are working properly.
```

File: quality-control/human-biomolecular-data.md [hunk 4]
```diff
@@ -1,27 +1,98 @@
...
+
+The mandatory and optional variables must be selected depending on the aim of the study and should include any relevant clinical data such as vaccination state, age, gender, or hospitalisation. More considerations about metadata can be found in the [Data resources page](https://www.infectious-diseases-toolkit.org/data-sources/human-biomolecular-data). 
+
+While completing the forms, the use of ontologies is highly recommended: 
+- Infectious Disease Ontology (IDO), a suite of interoperable ontology modules aiming to provide coverage of all aspects of the infectious disease domain. Further details about IDO can be found in this [paper](https://pubmed.ncbi.nlm.nih.gov/34275487/).
+- CovidO, an ontology for covid-19 metadata, is described in this [paper](https://link.springer.com/article/10.1007/s11227-023-05509-4).
+
+Besides sample metadata, assay metadata describing how the results were generated is also necessary. This metadata will depend on the type of assay (antigenic, molecular, target genes of the assay, serological…). And it should include all the used variables on which the results depend on.
```
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:94]:
```suggestion
Besides sample metadata, assay metadata describing how the results were generated is also necessary. This metadata will depend on the type of assay (antigenic, molecular, target genes of the assay, serological, etc) and it should include all the used variables on which the results depend on.
```

File: quality-control/human-biomolecular-data.md [hunk 5]
```diff
@@ -1,27 +1,98 @@
 ---
 title: Human biomolecular data
-description: Tools and methods to assess quality.
-contributors: []
-no_robots: true
-search_exclude: true
-sitemap: false
+description: This page explains in detail the importance of quality control for human biomolecular data in the context of infectious diseases, as well as explains the steps to follow.
+contributors: [Aina Jené Cortada, Mireia Marin Ginestar]
```
[the remark this answers is not in the document]
Comment (lauportell) [reply quality-control/human-biomolecular-data.md:4]: oh sorry that I missed them. Can you add Clementina to the Contributors yaml? I don't know her details.

File: quality-control/human-biomolecular-data.md [hunk 6]
```diff
@@ -1,27 +1,98 @@
...
+
+### Existing approaches
+
+When looking for solutions to sample collection and processing, you can check [this](https://rdmkit.elixir-europe.org/collecting) documentation.
+
+## Assessment of data quality and data preprocessing
+
+Pre-analytical errors, such as wrong labelling of the specimen, the incorrect specimen collection device, incorrect specimen type are the most frequently reported sources of error (between 48% and 62%) within the clinical laboratory, as stated in this 2021 [publication](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7917444/#s0120), Quality Assurance in the Clinical Virology Laboratory.
+
+Conducting quality control is an essential step to avoid these kinds of errors. Quality metrics such as read depth, coverage, and base quality scores can help identify errors, artifacts, and biases in the data. Tools such as FastQC, Qualimap, and Picard can be used to assess data quality and generate summary reports. On the other hand, tools such as the FastQScreen and Graf Pop and Sex, can be used to ensure the correct labelling of the samples.
```
[the remark this answers is not in the document]
Comment (lauportell) [reply quality-control/human-biomolecular-data.md]: done!

File: quality-control/human-biomolecular-data.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (lauportell) [reply quality-control/human-biomolecular-data.md]: Added! :)

**Comment by lauportell** [comment]
All comments are solved - I think this can be merged once it has a final review :)

**Review by bedroesb — APPROVED** [review]
Looks also good from a technical view!

**Comment by bedroesb** [comment]
@lauportell I noticed that there are some differences between the tool names in the name column vs when it is mentioned in its description. Could you quickly align them?

**Comment by bedroesb** [comment]
For example SAMtools vs Sam tools

**Pull request merged**